cs/GetMyWalletBalanceDemo.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net;
+using System.Net.Http;
```

Comment:
one needs to know that he needs to add System.Net.Http reference. It'd be nice to include that info in README, or provide a complete project that has all references preconfigured